libs/wingsdk/src/cloud/schedule.ts

```diff
@@ -0,0 +1,103 @@
+import { Construct } from "constructs";
+import { Polycons } from "polycons";
+import { Function, FunctionProps } from "./function";
+import { Code, IResource, Inflight, Resource } from "../core";
+import { Duration } from "../std";
+
+/**
+ * Global identifier for `Schedule`.
+ */
+export const SCHEDULE_TYPE = "wingsdk.cloud.Schedule";
+
+/**
+ * Properties for `Schedule`.
+ */
+export interface ScheduleProps {
+  /**
+   * Trigger events at a periodic rate.
+   * @example 1m
+   * @default undefined
+   */
+  readonly rate?: Duration;
+
+  /**
+   * Trigger events according to a cron schedule.
+   * @example "0/1 * ? * * *"
```

Comment:
I think we should add a link to the AWS schedule cron syntax since its not standard. Thoughts?
https://docs.aws.amazon.com/scheduler/latest/UserGuide/schedule-types.html#cron-based

Reply:
Yes, I agree, it is worth putting this information

Reply:
hmm yea I was not even thinking about this, it looks like even the major cloud providers dont agree on a standard cron format.
AWS:   { <minute> <hour> <day-of-month> <month> <day-of-week> <year> }
Azure: { <second> <minute> <hour> <day-of-month> <month> <day-of-week> }
GCP:   { <minute> <hour> <day-of-month> <month> <day-of-week> }

So basically AWS gives <year> and Azure gives <second>. @Chriscbr correct me if Im wrong but it looks like our spec uses the standard unix format (just like GCP). 

Since we dont want the cloud detail leaking to the user I think we should just offer the 5 fields (as it shows in the spec) and maybe update the SDK spec to clarify valid cron syntax. 

Then bear with me as the coffee needs to still kick in 😄 I also think maybe we might need to update the string depending on the target. 

@Chriscbr what about offering experience similar to CDK { minute: '30', hour: '2' }
https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_aws-events.CronOptions.html

Reply:
Cool, so @marciocadev lets change the references to use the 5 field format and for the AWS implementation we may just need to append a wildcard for year field. I imagine the API fails if you pass in only 5 fields versus the expected 6 but fact check me there 😁

Reply:
I think we can 'validate' the five value string by counting the spaces between them (4 blanks) at the moment and adding * at the end when deploying to AWS or * at the beginning when deploying to Azure.